### PR TITLE
Added first three batches of AM intro papers (more upcoming)

### PR DESCRIPTION
**Pull request by KirstensGitHub**
### List of citation keys added (one per line)

KahaEtal08b
BartEtal18
AnllHill96
HillMunt84
VaneEtal
LeonEtal
McadMaun00
PeirEtal19
MoorZirn17
PeteEtal19
UncaEtal11

### Other changes (describe)

### CODE REVIEW

**Comment by jeremymanning** [comment]
Need page numbers for all references, remove commas before "and", and use double hyphens to separate page ranges. Also remove months.  Author names should be {First name or initial} {Middle initial} {Last name}, with all authors separated by "` and `".

**Comment by KirstensGitHub** [comment]
changes pushed to my fork @jeremymanning ! New pull request needed?

File: memlab.bib [hunk 1]
```diff
@@ -110,6 +110,120 @@ @string{vn
 @string{vr = {Vision Research}}
 
 
+@article{cite-key,
+	Date-Added = {2019-07-10 10:37:42 +0000},
+	Date-Modified = {2019-07-10 10:37:48 +0000},
+	Title = {Interactions between space-based and feature-based attention.}}
+
+@article{BartEtal18,
+	Author = {Mandy V. Bartsch and Sarah E. Donohue and Hendrik Strumpf and Mircea A. Schoenfeld and Jens-Max Hopf},
```
Comment (jeremymanning) [inline memlab.bib]: remove "." after initials

File: memlab.bib [hunk 2]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Date-Added = {2019-07-10 10:34:24 +0000},
+	Date-Modified = {2019-07-10 17:44:29 +0000},
+	Journal = {Scientific Reports},
+	Number = {1},
+	Pages = {16132},
```
Comment (jeremymanning) [inline memlab.bib]: page range?

File: memlab.bib [hunk 3]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Title = {Enhanced spatial focusing increases feature-based selection in unattended locations},
+	Volume = {8},
+	Year = {2018}}
+
+@article{AnllHill96,
+	Author = {L Anllo-Vento and SA Hillyard},
```
Comment (jeremymanning) [inline memlab.bib]: separate "SA" into "S A"

File: memlab.bib [hunk 4]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Date-Modified = {2019-07-10 17:42:42 +0000},
+	Journal = {Perception and Psychophysics},
+	Number = {2},
+	Pages = {191--206},
+	Title = {Selective attention to the color and direction of moving stimuli: electrophysiological correlates of hierarchical feature selection.},
+	Volume = {58},
+	Year = {1996}}
+
+@article{HillMunt84,
+	Author = {Steven A. Hillyard and Thomas F. M{\"u}nte},
```
Comment (jeremymanning) [inline memlab.bib]: remove "."s after initials

File: memlab.bib [hunk 5]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Date-Modified = {2019-07-10 17:45:04 +0000},
+	Journal = {Perception and Psychophysics},
+	Number = {2},
+	Pages = {185--198},
+	Title = {Selective attention to color and location: An analysis with event-related brain potentials},
+	Volume = {36},
+	Year = {1984}}
+
+@article{VaneEtal18,
+	Author = {D.M. van Es and J. Theeuwes and T. Knapen},
```
Comment (jeremymanning) [inline memlab.bib]: remove "."s after initials, separate each initial with a space

File: memlab.bib [hunk 6]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Date-Added = {2019-07-10 10:15:27 +0000},
+	Date-Modified = {2019-07-10 17:46:11 +0000},
+	Journal = {eLife},
+	Pages = {e36928},
+	Title = {Spatial sampling in human visual cortex is modulated by both spatial and feature-based attention},
+	Volume = {7},
+	Year = {2018}}
+
+@article{LeonEtal,
+	Author = {CJ Leonard and A Balestreri and SJ Luck },
```
Comment (jeremymanning) [inline memlab.bib]: separate initials with spaces

File: memlab.bib [hunk 7]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Date-Added = {2019-07-10 10:13:25 +0000},
+	Date-Modified = {2019-07-10 17:47:02 +0000},
+	Journal = {Journal of Experimental Psychology: Human Perception \& Performance},
```
Comment (jeremymanning) [inline memlab.bib]: write out "and"

File: memlab.bib [hunk 8]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Number = {1},
+	Pages = {11--16},
+	Title = {Interactions between space-based and feature-based attention. Interactions between space-based and feature-based attention. Interactions},
```
Comment (jeremymanning) [inline memlab.bib]: typos in title (pasted several times?)

File: memlab.bib [hunk 9]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Volume = {41},
+	Year = {2015}}
+
+@article{McadMaun00,
+	Author = {CJ McAdams and JH Maunsell },
```
Comment (jeremymanning) [inline memlab.bib]: separate initials with spaces, remove trailing space after Maunsell

File: memlab.bib [hunk 10]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Date-Added = {2019-07-10 10:05:10 +0000},
+	Date-Modified = {2019-07-10 17:47:28 +0000},
+	Journal = {Journal of neurophysiology},
+	Number = {3},
+	Pages = {1751--1755},
+	Title = {Attention to both space and feature modulates neuronal responses in macaque area V4.},
```
Comment (jeremymanning) [inline memlab.bib]: no period at end of title

File: memlab.bib [hunk 11]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Volume = {83},
+	Year = {2000}}
+
+@article{PeirEtal19,
+	Author = {J W Peirce and JR Gray and S Simpson and MR MacAskill and R H{\"o}chenberger and H Sogo and E Kastman and J Lindel{\o}v},
```
Comment (jeremymanning) [inline memlab.bib]: separate all initials with spaces

File: memlab.bib [hunk 12]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Volume = {51},
+	Year = {2019}}
+
+@article{MoorZirn17,
+	Author = {Tirin Moore and Marc Zirnsak},
+	Date-Added = {2019-07-10 09:51:51 +0000},
+	Date-Modified = {2019-07-10 18:03:11 +0000},
+	Journal = {Annual Review of Psychology},
+	Pages = {47--72},
+	Title = {Neural Mechanisms of Selective Visual Attention},
```
Comment (jeremymanning) [inline memlab.bib]: sentence case

File: memlab.bib [hunk 13]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Volume = {68},
+	Year = {2017}}
+
+@article{PeteEtal19,
+	Author = {M S Peterson and S P Kelly and E J Blumberg },
+	Date-Added = {2019-07-10 09:48:59 +0000},
+	Date-Modified = {2019-07-10 18:04:23 +0000},
+	Journal = {Journal of Experimental Psychology: Human Perception \& Performance},
```
Comment (jeremymanning) [inline memlab.bib]: type out "and"

File: memlab.bib [hunk 14]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Number = {2},
+	Pages = {255--263},
+	Title = {Saccadic eye movements smear spatial working memory.},
+	Volume = {45},
+	Year = {2019}}
+
+@article{UncaEtal11,
+	Author = {Melina R. Uncapher and J. Benjamin Hutchinson and Anthony D. Wagner},
```
Comment (jeremymanning) [inline memlab.bib]: remove periods after initials

File: memlab.bib [hunk 15]
```diff
@@ -110,6 +110,120 @@ @string{vn
...
+	Date-Added = {2019-07-10 09:42:20 +0000},
+	Date-Modified = {2019-07-10 18:04:54 +0000},
+	Journal = {The Journal of Neuroscience},
+	Number = {35},
+	Pages = {12613--12628},
+	Read = {0},
+	Title = {Dissociable Effects of Top-Down and Bottom-Up Attention during Episodic Encoding},
```
Comment (jeremymanning) [inline memlab.bib]: sentence case

File: memlab.bib [hunk 16]
```diff
@@ -9162,6 +9276,7 @@ @article{FaraMcCl91
 @article{LyonEtal08,
 	Abstract = {Visualizing spatial material is a cornerstone of human problem solving, but human visualization capacity is sharply limited. To investigate the sources of this limit, we developed a new task to measure visualization accuracy for verbally-described spatial paths (similar to street directions), and implemented a computational process model to perform it. In this model, developed within the Adaptive Control of Thought-Rational (ACT-R) architecture, visualization capacity is limited by three mechanisms. Two of these (associative interference and decay) are longstanding characteristics of ACT-R's declarative memory. A third (spatial interference) is a new mechanism motivated by spatial proximity effects in our data. We tested the model in two experiments, one with parameter-value fitting, and a replication without further fitting. Correspondence between model and data was close in both experiments, suggesting that the model may be useful for understanding why visualizing new, complex spatial material is so difficult.},
```
Comment (jeremymanning) [inline memlab.bib]: remove abstract

**Comment by jeremymanning** [comment]
closing this request...feel free to reopen if needed

**Pull request closed without merging**